packages/core/src/App/Components/Layout/Header/dtrader-v2/account-actions-dtrader-v2.tsx

```diff
@@ -27,17 +28,6 @@ export type TAccountInfoDTraderV2 = Omit<
     is_disabled?: boolean;
     toggleDialog: (value?: boolean | undefined) => void;
 };
-const AccountInfoDTraderV2 = React.lazy(
-    () =>
-        moduleLoader(
-            () =>
-                import(
-                    /* webpackChunkName: "account-info-dtrader-v2", webpackPreload: true */ 'App/Components/Layout/Header/dtrader-v2/account-info-dtrader-v2'
-                )
-        ) as Promise<{
-            default: React.ComponentType<TAccountInfoDTraderV2>;
-        }>
-);
```

Comment:
Lazy loading of the header causes some quill styles in the core to have more specificity than the dtrader styles and it causes style issues in the purchase button.